base/shell/explorer/lang/ro-RO.rc

```diff
@@ -79,7 +79,7 @@ BEGIN
     AUTOCHECKBOX "&Menține bara de activități deasupra celorlalte ferestre", IDC_TASKBARPROP_ONTOP, 13, 71, 200, 10
     AUTOCHECKBOX "&Grupează butoanele similare din bara de activități", IDC_TASKBARPROP_GROUP, 13, 84, 200, 10
     AUTOCHECKBOX "Afișează bara „&Lansări rapide”", IDC_TASKBARPROP_SHOWQL, 13, 97, 200, 10
-    AUTOCHECKBOX "&Folosește iconițe mici", IDC_TASKBARPROP_SMALLICONS, 13, 110, 200, 10
+    AUTOCHECKBOX "Folo&sește pictograme mici", IDC_TASKBARPROP_SMALLICONS, 13, 110, 240, 10
```

Comment:
Assignment of access key on "S" is not good in Romanian, as it gets needlessly painted very close to "Ș" (s with comma below), see [Romanian translation notes, pt. №17](https://github.com/reactos/reactos/blob/master/media/doc/Romanian%20translation%20notes.txt#L41). Please change it to something else. The keys used in this dialog are "A", "G", "L", "M", and "X" (plus "F" and "N"), so given the text of this autocheckbox (and that "S", "Ș", "T", and "P" are not good candidates), the options that remain are "O", "E", "C", and "R".
Also, the access key on line №78 is overriding what should have been left for "Aplică". I suggest assigning something different for that checkbox too, like "U", or "D".